client/src/utils/polyfills.js

```diff
@@ -8,3 +8,7 @@ import 'core-js/shim';
 import 'whatwg-fetch';
 // IE11.
 import 'element-closest';
+// IE11.
+import svg4everybody from 'svg4everybody';
+
+svg4everybody();
```

Comment:
Should we wrap this in our `IS_IE11`? https://github.com/wagtail/wagtail/blob/882f8f3cf8ddd79c30e611a48882b309e90dad0c/client/src/config/wagtailConfig.js